packages/cicero-cli/lib/commands.js

```diff
@@ -496,7 +496,16 @@ class Commands {
     static archive(templatePath, target, outputPath, options) {
         return Commands.loadTemplate(templatePath, options)
             .then(async (template) => {
-                const archive = await template.toArchive(target);
+                let keystore = null;
+                if (options.keystore.path) {
+                    const p12File = fs.readFileSync(options.keystore.path, { encoding: 'base64' });
+                    const inputKeystore = {
+                        p12File: p12File,
+                        passphrase: options.keystore.passphrase
+                    };
+                    keystore = inputKeystore;
+                }
+                const archive = await template.toArchive(target, {keystore});
```

Comment:
This means only the `keystore` option is being passed. Other options such as `warnings: true` should not be dropped.